Report (#1)
inbo/figquest-report: source/trendfiguur/04_conclusies.Rmd

# Conclusies

Deelnemers aan deze studie gaven hun voorkeur aan voor trendfiguren.
Daarnaast lieten we hen twee reeksen van drie trendfiguren interpreteren.
De eerste reeks was de meest eenvoudige figuur met de schaal van de y-as als een index.
De tweede reeks was in de voorkeursstijl van de deelnemer.

**Globale voorkeur**

De deelnemers hadden een lichte voorkeur voor een y-as met een index in plaats van een procentuele wijziging.
Het merendeel vond de referentielijnen een meerwaarde.
De annotatie van deze lijnen was eveneens belangrijk.
Driekwart van de deelnemers koos voor de weergave van de onzekerheid, bij voorkeur met een eenvoudig band in plaats van een gradiënt.
De indeling van de trends met behulp van symbolen werd door een derde van de deelnemers gesmaakt.

> Deelnemers die aangaven minder vertrouwd te zijn met cijfers en statistiek kiezen iets vaker voor een eenvoudigere figuur zonder referentielijnen en onzekerheid en met een index in plaats van procentuele wijziging op de y-as.

Reviewer: ```suggestion
Deelnemers die aangaven minder vertrouwd te zijn met cijfers en statistiek, kiezen iets vaker voor een meer eenvoudige figuur zonder referentielijnen en onzekerheid en met een index in plaats van procentuele wijziging op de y-as.
```

> Deelnemers die aangaven minder vertrouwd te zijn met cijfers en statistiek kiezen iets vaker voor een eenvoudigere figuur zonder referentielijnen en onzekerheid en met een index in plaats van procentuele wijziging op de y-as.

Reviewer: Dus ergens is het misschien toch beter om de figuren eenvoudig te houden voor een breed publiek?

Nadeel van uw proefopzet is wel dat je op die manier niet gaat kunnen controleren of ze een figuur met referentielijnen en onzekerheid wel correct zouden kunnen interpreteren.  Ideetje voor een volgende enquête is misschien om achteraf de resultaten nog eens te laten interpreteren met referentielijn en/of onzekerheid bij?  Of een nieuwe studie met focus op de vraag in hoeverre een breed publiek bepaalde types grafieken correct kan interpreteren: de resultaten laten interpreteren met enkele verschillende grafieken: bv. met referentielijn en/of met onzekerheid,...  Evt. als laatste vraag hun voorkeur vragen.

Qua voorkeur voor symbolen was er geen verschil tussen de types deelnemers.

**Effect op correcte inschatting**

Beide reeksen figuren die we de deelnemers voorschotelden, hadden dezelfde onderliggende gegevens.
Ze verschilden enkel in de manier van weergave.
Hierdoor kunnen we een eerste inschatting maken van het effect van de weergave op de mogelijkheid om de figuur juist te interpreteren.
Aangezien meer dan 90% van de deelnemers zowel een betrouwbaarheidsinterval wensten als bijkomende referentielijnen, kunnen we deze effecten niet van elkaar onderscheiden.
We hebben ze daarom samen genomen als het effect van de tweede reeks.

> Bij de eerste reeks met de eenvoudige figuur gaven slechts vier van de tien deelnemers een correcte interpretatie van de figuur.

Reviewer: Niet akkoord dat de interpretatie niet correct is als er te weinig info aanwezig is om de grafiek correct te interpreteren.  In zo'n geval is 'onduidelijk' of 'weet ik niet' volgens mij minstens even correct (of misschien zelfs correcter?) dan gokken wat een belangrijke wijziging zou zijn, of hoe groot het betrouwbaarheidsinterval zou zijn...

> We stelden hierbij vooral een overinterpretatie van de trendfiguren vast: de deelnemers hadden de neiging om niet significante effecten toch als een stijging te interpreteren.

Reviewer 2: ```suggestion
We stelden hierbij vooral een overinterpretatie van de trendfiguren vast: de deelnemers hadden de neiging om niet significante effecten toch als een significant te interpreteren.
```

> We stelden hierbij vooral een overinterpretatie van de trendfiguren vast: de deelnemers hadden de neiging om niet significante effecten toch als een significant te interpreteren.

Reviewer: Is er eigenlijk ook gecheckt of mensen met kennis van statistiek 'voorzichtiger' zijn geweest (zelfs ingeval van significante effecten)?

Bij de tweede reeks was driekwart van de antwoorden correct.

> Hou hierbij in het achterhoofd dat de deelnemers bij deze reeks meestal zowel het betrouwbaarheidsinterval als de referentielijnen te zien kregen.

Reviewer: Wat met de deelnemers die geen van beiden te zien kregen?

Tijdens de tweede reeks was er veel minder overinterpretatie van de figuren.
De meest voorkomende verkeerde interpretaties waren onduidelijk in plaats van vergelijkbaar en omgekeerd.

**Aanbevelingen voor verder onderzoek**

Het is wenselijk om in de toekomst meer deelnemers aan te trekken die zich minder vertrouwd voelen met cijfers en of statistiek.

> Gezien het onderwerp zal geen eenvoudige klus zijn.

Reviewer 2: ```suggestion
Gezien het onderwerp zal dat geen eenvoudige klus zijn.
```

> Om het effect van betrouwbaarheidsintervallen en referentielijnen van elkaar te kunnen onderscheiden, moeten we een aangepaste vragenlijst opstellen.

Reviewer: Zie ook mijn voorstel hierboven: gewoon de verschillende soorten grafieken voorschotelen: enkele met betrouwbaarheidsintervallen, enkele met referentielijnen,... en testen wat correct ingeschat wordt.  Evt. kan je afsluiten met het vragen naar de voorkeuren.  (Met enkel een test naar interpretatie en zonder die voorkeuren gaat de enquête hopelijk ook wat korter worden en wat behapbaarder zijn voor niet-kenners?)
